src/components/SpruceForm/LeafyGreenWidgets.tsx

```diff
@@ -54,6 +55,7 @@ export const LeafyGreenTextInput: React.FC<WidgetProps> = ({
           label={ariaLabelledBy ? undefined : label}
           placeholder={placeholder || undefined}
           description={description as string}
+          optional={optional as boolean}
```

Comment:
I noticed a default CSS rule is making this look weird. We can reset the margins here by applying this styling to the input.

```js
const StyledTextInput = styled(TextInput)`
  p {
    margin: 0;
  }
`;
```

Reply:
thank you!